dashboard/public/locales/en/translation.yaml

```diff
@@ -0,0 +1,1463 @@
+getStarted:
```

Comment:
This file is not required, we are not using translation in dashboards, right? Also this file is for Portal not for dahsboard.